webapp/channels/src/components/post_view/post_flag_icon/post_flag_icon.tsx

```diff
@@ -19,115 +19,97 @@ export type Actions = {
     unflagPost: typeof unflagPost;
 }
 
-interface Props {
+type Props = {
```

Comment:
why the change here

Reply:
I was receiving a weird typescript error (maybe just on local) where the index file was complaining with the following error: `Default export of the module has or is using private name 'Props'.`. Moving it to a type fixed it.

Reply:
hmm thats interesting